src/data/customer/Job.js

```diff
@@ -52,6 +56,8 @@ export const getJobSummaryAsync = async (jobId, token) => {
  * @param {string} token
  * @return {object} Promise resolve/reject
  */
+ //TODO: needs unit testing
+ //TODO: there no such thing as Recommended Jobs? Should be deprecated?
```

Comment:
This is actually used to get the job detail(not summary), and **RECOMMENDED_JOB** used to have same value as **JOB**, suggest to rename this to getJobDetailsAsync or something like that and update the URL endpoint to endpoints.API_V3.JOB

Reply:
Really, because I can't find the function in the react codebase and we have a Job summary function on job.js?@ccwukong